### PR TITLE
Shorter more focused readme

### PR DESCRIPTION
**Pull request by hpk42**
- remove feature descrpition
- focus on build instructions 
- shorten some section (eg don't explain GPL)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,69 +2,19 @@
...
---------------------------------------------------------------------------------
-
-The C-API is documented at <https://deltachat.github.io/api/>.
-
-Please keep in mind, that your derived work must be released under a
-GPL-compatible licence.  For details, please have a look at the [LICENSE file](https://github.com/deltachat/deltachat-core/blob/master/LICENSE) accompanying the source code.
+The _Delta Chat Core Library_ is written in cross-platform **C**,
+documented at <https://deltachat.github.io/api/>.
 
+Building the C-library
```
Comment (ralphtheninja) [inline README.md]: Can we do `#` etc for different headers?

File: README.md [hunk 2]
```diff
@@ -130,10 +108,13 @@ You start the program with `./delta <database-file>`
...
+https://github.com/deltachat/deltachat-core/tree/master/python/tests
+
 
 License
 --------------------------------------------------------------------------------
 
 Licensed under the GPLv3, see [LICENSE](./LICENSE) file for details.
 
-Copyright © 2017, 2018 Delta Chat contributors.
+Copyright © 2017, 2018 Bjoern Petersen and Delta Chat contributors.
```
Comment (ralphtheninja) [inline README.md]: Björn Petersen

**Comment by flub** [comment]
Nice!  LGTM

**Comment by r10s** [comment]
build in @ralphtheninja comments, lgtm, merging :)

**Pull request merged**